recipes/ltbams/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "ltbams" %}
+{% set version = "0.8.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ltbams-{{ version }}.tar.gz
+  sha256: e99095153da21662e4b3ab930177044a8088a62fae21f9e7ed7563732416057a
+
+build:
+  entry_points:
+    - ams = ams.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

Comment:
Does this package require a c++ compiler to build an extension module? I don't see any in the github source. If it doesn't, please remove this build requirement. If there is a c++ extension, then this shouldn't be `noarch: python`.

Reply:
Thanks for the feedback, let me double check this.

Reply:
After check, there is no dependency on c++ extension, so I've removed the build requirement.
Thanks again for the feedback!